### ISSUE TITLE
Screen area where upload is not executed, after D&D to URL shared link screen on the browser

### ISSUE THREAD

**Issue by tatsuyasenbom** [nextcloud/server#28974]
### Overview
* Screen area where upload is not executed, after D&D(dragging and dropping) to URL shared link screen on the browser.

### Steps to reproduce & Actual behaviour
1. Create a URL shared link of folder.
2. Access to URL shared link by browser.
3. When I D&D to the screen area between  `<div id="files-public-content">` and `<footer>`(Red ellipse
), the upload is not executed.
    * ![image](https://user-images.githubusercontent.com/87968771/135015230-77361716-72c6-49c9-be71-3fe24aeea611.png)

### Expected behaviour
* After D&D to between `<div id="files-public-content">` and `<footer>`, I expect the upload executed.

### My temporary workaround
* After making the following corrections, I confirmed that the phenomenon was improved.

```sh
# vim /var/www/nextcloud/apps/files_sharing/templates/public.php
:
        <!-- files listing -->
        <div id="files-public-content" style="height: 100%;">
:
```

### Server configuration

```sh
# cat /etc/redhat-release 
CentOS Linux release 7.9.2009 (Core)
# yum info httpd
:
名前                : httpd
アーキテクチャー    : x86_64
バージョン          : 2.4.6
リリース            : 97.el7.centos
:
# mysql --version
mysql  Ver 15.1 Distrib 10.5.12-MariaDB, for Linux (x86_64) using readline 5.1
# php --version
PHP 7.3.29 (cli) (built: Jun 29 2021 09:30:31) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.3.29, Copyright (c) 1998-2018 Zend Technologies
    with Zend OPcache v7.3.29, Copyright (c) 1999-2018, by Zend Technologies
[root@skycloud nextcloud]# cat /var/www/nextcloud/version.php 
<?php 
$OC_Version = array(22,1,1,2);
$OC_VersionString = '22.1.1';
$OC_Edition = '';
$OC_Channel = 'stable';
$OC_VersionCanBeUpgradedFrom = array (
  'nextcloud' => 
  array (
    '21.0' => true,
    '22.0' => true,
    '22.1' => true,
  ),
  'owncloud' => 
  array (
    '10.5' => true,
  ),
);
$OC_Build = '2021-08-26T13:27:46+00:00 1eea64f2c3eb0e110391c24830cea5f8d9c3e6a1';
$vendor = 'nextcloud';
```

**Comment by szaimen** [comment]
Hi, please update to 24.0.9 or better 25.0.3 and report back if it fixes the issue. Thank you!

My goal is to add a label like e.g. 25-feedback to this ticket of an up-to-date major Nextcloud version where the bug could be reproduced. However this is not going to work without your help. So thanks for all your effort!

If you don't manage to reproduce the issue in time and the issue gets closed but you can reproduce the issue afterwards, feel free to create a new bug report with up-to-date information by following this link: https://github.com/nextcloud/server/issues/new?assignees=&labels=bug%2C0.+Needs+triage&template=BUG_REPORT.yml&title=%5BBug%5D%3A+

**Comment by tatsuyasenbom** [comment]
> 3\. files-public-content

Thank you for contacting me!
I have confirmed the issue has been resolved in v25.0.3!

![image](https://user-images.githubusercontent.com/87968771/214187726-1a9be951-0dec-46d6-b989-ec70c31f0527.png)
